packages/terra-site/src/examples/tag/TagIconAndText.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import IconBookmark from 'terra-icon/lib/icon/IconTile';
+import Tag from 'terra-tag';
+
+const Icon = <IconBookmark />;
+
+const TagIconAndText = () => (
+  <div>
+    <Tag icon={Icon} text="Icon And Text Tag" />
+    <Tag icon={Icon} onClick={() => 1} text="Icon And Text Tag onClick" />
```

Comment:
Do the alert here as well, and the other examples that have an onClick

Reply:
Added it [here](https://github.com/cerner/terra-core/pull/1306/commits/b04c1661bc417ddbf31b7da23c28d95ec857e1df).